donkeycarManagerFront/src/js/service.js

```diff
@@ -228,14 +233,14 @@ export default class DonkeycarManagerService {
     // Drive Job
     const driveJob = Object.assign({
       name: 'DRIVE',
-      parameters: JSON.stringify({ drive_time: 10 }, null, 2), // 30 sec of driving session
+      parameters: JSON.stringify({ drive_time: parseInt(driveTime) }, null, 2), // 30 sec of driving session
```

Comment:
why is driveTime a string ? you should parseInt in the code that calls this function not here

Reply:
tutorial example

Reply:
we can't store int values : https://stackoverflow.com/questions/33952287/is-it-possible-to-store-integer-value-in-localstorage-like-in-javascript-objects

Reply:
ooh wait yes